src/main/scala/wdl4s/expression/FileEvaluator.scala

```diff
@@ -115,7 +115,7 @@ case class FileEvaluator(valueEvaluator: ValueEvaluator, coerceTo: WdlType = Wdl
       case a: Ast if a.isArrayOrMapLookup =>
         evalValue(a) match {
           case Success(f: WdlFile) => Success(Seq(f))
-          case _ => evaluateRecursive(a.getAttribute("rhs"))
+          case _ => evaluateRecursive(a.getAttribute("lhs"))
```

Comment:
Centaur last night said it was fine. You'll have to take my offline word for it though, as- as far as I know- we don't have a way of travis testing wdl4s PRs.